pkg/operator/controllers/genevalogging/genevalogging.go

```diff
@@ -69,6 +70,13 @@ func (g *genevaLogging) securityContextConstraints(ctx context.Context, name, se
 }
 
 func (g *genevaLogging) daemonset(r azure.Resource) *appsv1.DaemonSet {
+
+	// TODO: remove once ACRDomain is populated in all clusters
+	acrDomain := g.cluster.Spec.ACRDomain
+	if acrDomain == "" {
+		acrDomain = g.cluster.Spec.ACRName + ".azurecr.io"
+	}
```

Comment:
@jim-minter This will allow us to use ACRDomain if present, otherwise fall back to ACRName.

Reply:
I don't believe this should be necessary, but it's OK that it's there I guess.  Just remember to remove it later.